roles/docker/tasks/debian.yml

```diff
@@ -1,26 +1,34 @@
 ---
+- name: Ensure old versions of Docker are not installed.
```

Comment:
Since all other tasks' names don't start with a captial letter and not using a period at the end, I think it is better to keep the same style here.